core/src/apps/debug/__init__.py

```diff
@@ -138,6 +139,27 @@ def notify_layout_change(layout: ui.Layout) -> None:
             crypto.random.reseed(msg.value)
         return Success()
 
+    async def dispatch_DebugLinkEraseSdCard(
+        ctx: wire.Context, msg: DebugLinkEraseSdCard
+    ) -> Success:
+        try:
+            io.sdcard.power_on()
+            if msg.format:
+                fs = io.FatFS()
+                fs.mkfs()
+            else:
+                # trash first 1 MB of data
```

Comment:
Why's that? What does it mean exactly?

Reply:
I don't think I understand your question?

the purpose of this debug message is to present a "clean" SD card, either cleanly-formatted, or cleanly-unformatted. Trashing the first 1MB seems a safe way to destroy the FAT filesystem

Reply:
I see, I did not realize. Could we change the comment to "Trash the first 1MB to destroy the FAT filesystem"?

Reply:
done in rebase